src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -5525,6 +5229,7 @@ utimens_timespec(const char *path, const struct timespec times[2], int flags)
 		times_us[0].tv_usec = times[0].tv_nsec / 100;
 		times_us[1].tv_sec  = times[1].tv_sec;
 		times_us[1].tv_usec = times[1].tv_nsec / 100;
+		drec_decref(dfs_mt->dcache, parent);
```

Comment:
This line should be removed since "is_target_path" is false here.

Reply:
Agree, that this statement is useless but should be harmless.
Indeed `parent` will be `NULL` and thus `drec_decref()` will do nothing (as for the `free()` function with NULL pointer).
Thus, if it is OK for you, I will fix this in the follow-up PR.
Is it OK for you.

Reply:
yes let's fix that in the GC follow on PR please

Reply:
Sounds good to me.

Reply:
Fixed in the GC follow-up PR with commit ece16c1146b4eeba9f6ec2769b558c2abcdda989